packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -129,6 +129,7 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
       crawlerGettingStarted: `${ENTERPRISE_SEARCH_DOCS}crawler-getting-started.html`,
       crawlerManaging: `${ENTERPRISE_SEARCH_DOCS}crawler-managing.html`,
       crawlerOverview: `${ENTERPRISE_SEARCH_DOCS}crawler.html`,
+      deployTrainedModels: `${ELASTIC_WEBSITE_URL}guide/en/machine-learning/${DOC_LINK_VERSION}/ml-nlp-deploy-models.html`,
```

Comment:
Should we create `MACHINE_LEARNING_DOCS`, which encapsulates `${ELASTIC_WEBSITE_URL}guide/en/machine-learning/${DOC_LINK_VERSION}` ?

I ask because this entry stands out as different from the others.

Reply:
I think this entry is a duplicate of `startTrainedModelsDeployment` and therefore redundant?

Reply:
Makes sense to me. There is a whole bunch of other links on the same file from L402 to L429. 
To make sure I test it throughly, I will create the `${MACHINE_LEARNING_DOCS}` but merge them in a different PR (following up this one) that aims for 8.6 instead of 8.5

Reply:
@chriscressman Yes, it's true that we could add a new shortcut for the ML Guide, but it's not necessary in this PR IMO. Each time a new shortcut like that is created, we also have to update the doc build to be aware of it, so it would require a matching PR in the docs repo.

Reply:
@chriscressman Good idea to add the shortcut for ML! I took note of it and will add the shortcut soon and handle the docs repo PR, too, if it's okay with you.

Reply:
All sounds good to me! I agree we don't need the change on this PR and can handle it separately if others think that's useful.

Reply:
I added the shortcut via https://github.com/elastic/kibana/pull/142091 and https://github.com/elastic/docs/pull/2534. Thanks for your patience!